ApollosProject/apollos-prototype
# Sets new phone numbers in SMS Auth as Mobile phone numbers

## DESCRIPTION

![2019-06-26 07 54 05](https://user-images.githubusercontent.com/1637694/60178267-1363ea00-97e9-11e9-9ba5-59e611d4ee51.gif)

### What does this PR do, or why is it needed?

When creating a new user, with a phone number, we should make sure that number is setup as their Mobile number. This PR accomplishes that. 

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

1. Login with a phone number you have never logged in before.
 - Or manually delete the UserLogin with your phone number as the UserName
2. Ensure that after logging in via SMS, your phone number is listed as a mobile number

### What automated tests did you write?

- Updated existing tests.

## TODO:

- [x] Closes #823 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
